edk2toollib/uefi/edk2/parsers/dec_parser.py

```diff
@@ -199,6 +200,17 @@ def _Parse(self) -> None:
             elif InPcdSection:
                 if sline.strip()[0] == '[':
                     InPcdSection = False
+                elif sline.strip().endswith("{"):
```

Comment:
Is it sufficient to only check the end of the line? Are '{' in the middle of a line a concern? Or has it been preprocessed by this point in the code?

Reply:
It will be pre-processed, so comments will be stripped and blank lines skipped

Reply:
There's no possibility of an inline struct or something? Like...
```
PcdSomethingSomething|{blah, blah}
```

Reply:
That's a great point.

Reply:
Maybe just look at if the count of({) > count of (})

Reply:
Dunno. Just trying to poke holes in your work. :-p

Reply:
No it's good :)